task3/server/server.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const fs = require('fs');
+const bodyParser = require('body-parser');
+const { v4: uuidv4 } = require('uuid');
+
+const port = 3000;
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.listen(port, () => {
+  console.log('Server is running');
+});
+
+app.get('/users', (req, res) => {
+  fs.readFile('./task3/db/users.json', 'utf-8', (err, data) => {
+    if(err) console.log("error: ", err);
+    const users = JSON.parse(data);
+    res.send(users);
+  });
+});
+
+app.get('/users/:id', (req, res) => {
+  let { id } = req.params;
+  fs.readFile('./task3/db/users.json', 'utf-8', (err, data) => {
+    if(err) console.log("error: ", err);
+    const users = JSON.parse(data);
+    const user = users.filter(user => user.id === id);
+    res.send(user);
+  });
+});
+
+app.post('/users', (req, res) => {
+  let { user } = req.body;
+  const id = uuidv4();
```

Comment:
is it make sense to use here uuid if users which you already have, have ids 1,2,3,4...